# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1232667225 18000
#      Thu Jan 22 18:33:45 2009 -0500
# Node ID ab61b47d343584a12e9a7be0014eee834daed91f
# Parent  6f0e539b3fc389296e3fabcf6c1b4d162cc4550d
configure.in (AH_BOTTOM): Also check __MACH__ when defining OCTAVE_USE_OS_X_API

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,12 +1,12 @@
 2009-01-22  John W. Eaton  <jwe@octave.org>
 
 	* configure.in (AH_BOTTOM): Define OCTAVE_USE_OS_X_API if
-	__APPLE__ is defined.
+	__APPLE__ and __MACK__ are defined.
 
 2009-01-22  Jaroslav Hajek  <highegg@gmail.com>
 
 	* configure.in: Fix qrupdate warning message.
 
 2009-01-21  John W. Eaton  <jwe@octave.org>
 
 	* Makeconf.in: Substitute X11_INCFLAGS and X11_LIBS.
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -1982,17 +1982,17 @@ typedef int sig_atomic_t;
 #define OCTAVE_HAVE_POSIX_FILESYSTEM 1
 #endif
 
 /* Define if we expect to have <windows.h>, Sleep, etc. */
 #if defined (__WIN32__) && ! defined (__CYGWIN__)
 #define OCTAVE_USE_WINDOWS_API 1
 #endif
 
-#if defined (__APPLE__)
+#if defined (__APPLE__) && defined (__MACH__)
 #define OCTAVE_USE_OS_X_API 1
 #endif
 
 /* sigsetjmp is a macro, not a function. */
 #if defined (sigsetjmp) && defined (HAVE_SIGLONGJMP)
 #define OCTAVE_HAVE_SIG_JUMP
 #endif
 
